packages/frontend/amp/components/Sidebar.tsx

```diff
@@ -198,15 +198,20 @@ const template = `
 `;
 
 // tslint:disable:react-no-dangerous-html
-export const Sidebar: React.FC<{ nav: NavType }> = ({ nav }) => (
-    <amp-sidebar class={sidebarStyles} layout="nodisplay" id="sidebar1">
-        <amp-list
-            layout="fill"
-            src="https://amp.theguardian.com/editionalised-nav.json"
-        >
-            <template type="amp-mustache">
-                <div dangerouslySetInnerHTML={{ __html: template }} />
-            </template>
-        </amp-list>
-    </amp-sidebar>
-);
+export const Sidebar: React.FC<{ nav: NavType }> = ({ nav }) => {
+    // this next line is necessary cos react has a 'template' object with no 'type' property.
+    // By saying 'as {}' we can pretend we're not adding the 'type' property and thus avoid unhappy type errors
```

Comment:
What is the advantage of this over the previous `template: any` override?

Reply:
`template: any` isn't possible any more as 'template' has become a thing in react

Reply:
I think this is as a result of the typescript or tsline upgrades, I'm not sure